rpc/client/rpc_test.go

```diff
@@ -576,10 +576,10 @@ func TestTxSearch(t *testing.T) {
 		assert.True(t, ptx.TxResult.IsOK())
 		assert.EqualValues(t, find.Hash, ptx.Hash)
 
-		// time to verify the proof
-		if assert.EqualValues(t, find.Tx, ptx.Proof.Data) {
-			assert.NoError(t, ptx.Proof.Proof.Verify(ptx.Proof.RootHash, find.Hash))
-		}
+		// // time to verify the proof
+		// if assert.EqualValues(t, find.Tx, ptx.Proof.Data) {
```

Comment:
same, are proofs not yet supported?

Reply:
i commented it out, but kept it to minimize diffs later. this feature is hardcoded to not work on mainnet.